ordersys/funcs/operate.py

```diff
@@ -46,3 +47,53 @@ def one_click_order(user, **data):
         OrderProductTypeBind.objects.create(oid=order, **dic)
 
     return order
+
+
+@user_from_sid(Error404)
+def compete_order(user, oid):
+    order = OrderInfo.objects.filter(id=oid).first()
```

Comment:
这里如果order is None服务器会报错500 AttributeError。建议：
```suggestion
    try:
        order = OrderInfo.objects.get(id=oid)
    except OrderInfo.DoesNotExist:
        raise WLException(404, u"订单不存在")
```
其他文件请做类似改动。